test/helm/1clonevol/templates/pvc0.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: restorepvc
-  namespace: test
+  namespace: namespace: {{ .Values.namespace }}
```

Comment:
you have typed the variable namespace twice

Reply:
fixed -- thanks